operator/internal/controller/redpanda/decommission-role.yaml

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
Where did this file come from?

Reply:
My mistake. At first I was trying to copy `decommission-role.yaml` to solve rpk bundle permissions that are missing in redpanda package integration test suite. Then I realised the fix for cloud is orthogonal to our broken setup of integration test suite. I lean towards land cloud fix sooner and create follow up to fix integration test suite.